From patchwork Tue Mar 30 19:30:29 2021
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Emanuele Giuseppe Esposito <eesposit@redhat.com>
X-Patchwork-Id: 12173599
Return-Path: <kvm-owner@kernel.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-15.8 required=3.0 tests=BAYES_00,DKIMWL_WL_HIGH,
	DKIM_SIGNED,DKIM_VALID,DKIM_VALID_AU,HEADER_FROM_DIFFERENT_DOMAINS,
	INCLUDES_CR_TRAILER,INCLUDES_PATCH,MAILING_LIST_MULTI,SPF_HELO_NONE,SPF_PASS
	autolearn=unavailable autolearn_force=no version=3.4.0
Received: from mail.kernel.org (mail.kernel.org [198.145.29.99])
	by smtp.lore.kernel.org (Postfix) with ESMTP id 3E752C433C1
	for <kvm@archiver.kernel.org>; Tue, 30 Mar 2021 19:31:47 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id F21CD6148E
	for <kvm@archiver.kernel.org>; Tue, 30 Mar 2021 19:31:46 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S233121AbhC3TbT (ORCPT <rfc822;kvm@archiver.kernel.org>);
        Tue, 30 Mar 2021 15:31:19 -0400
Received: from us-smtp-delivery-124.mimecast.com ([170.10.133.124]:37844 "EHLO
        us-smtp-delivery-124.mimecast.com" rhost-flags-OK-OK-OK-OK)
        by vger.kernel.org with ESMTP id S233161AbhC3Tao (ORCPT
        <rfc822;kvm@vger.kernel.org>); Tue, 30 Mar 2021 15:30:44 -0400
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=redhat.com;
        s=mimecast20190719; t=1617132643;
        h=from:from:reply-to:subject:subject:date:date:message-id:message-id:
         to:to:cc:cc:mime-version:mime-version:
         content-transfer-encoding:content-transfer-encoding;
        bh=eTWTr8cAwYev5yD8LuZQ3faxQx+nRJ4mRBG0w/bSeiQ=;
        b=JTfJeicYClaN2cgzqjMKbs0a+/dOjyVckVZVTj8mQiH+GtZ4TLWrVPr3SMHHhrffIildj+
        5zKkCf4kSIkXGjFyudphX/osJgZQTOcJGcKUQzbaMSNISczlWjMdmQKsWm2CK1XxinfGz8
        VwpiazQ+BTzoIfp2rHPq5ALD61qUr7g=
Received: from mimecast-mx01.redhat.com (mimecast-mx01.redhat.com
 [209.132.183.4]) (Using TLS) by relay.mimecast.com with ESMTP id
 us-mta-428-VdVI23fzMqqwbqr38R4dmQ-1; Tue, 30 Mar 2021 15:30:39 -0400
X-MC-Unique: VdVI23fzMqqwbqr38R4dmQ-1
Received: from smtp.corp.redhat.com (int-mx08.intmail.prod.int.phx2.redhat.com
 [10.5.11.23])
        (using TLSv1.2 with cipher AECDH-AES256-SHA (256/256 bits))
        (No client certificate requested)
        by mimecast-mx01.redhat.com (Postfix) with ESMTPS id 68F069CDA4;
        Tue, 30 Mar 2021 19:30:38 +0000 (UTC)
Received: from localhost.localdomain.com (ovpn-112-41.ams2.redhat.com
 [10.36.112.41])
        by smtp.corp.redhat.com (Postfix) with ESMTP id 99F9719C44;
        Tue, 30 Mar 2021 19:30:36 +0000 (UTC)
From: Emanuele Giuseppe Esposito <eesposit@redhat.com>
To: kvm@vger.kernel.org
Cc: Paolo Bonzini <pbonzini@redhat.com>,
        Vitaly Kuznetsov <vkuznets@redhat.com>,
        Andrew Jones <drjones@redhat.com>,
        linux-kernel@vger.kernel.org, linux-kselftest@vger.kernel.org
Subject: [PATCH] kvm: fix minor typos in x86/kvm.h and selftests/processor.c
Date: Tue, 30 Mar 2021 21:30:29 +0200
Message-Id: <20210330193029.47746-1-eesposit@redhat.com>
MIME-Version: 1.0
X-Scanned-By: MIMEDefang 2.84 on 10.5.11.23
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Signed-off-by: Emanuele Giuseppe Esposito <eesposit@redhat.com>
---
 tools/arch/x86/include/uapi/asm/kvm.h              | 2 +-
 tools/testing/selftests/kvm/lib/x86_64/processor.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/arch/x86/include/uapi/asm/kvm.h b/tools/arch/x86/include/uapi/asm/kvm.h
index 5a3022c8af82..e00d44bc5f55 100644
--- a/tools/arch/x86/include/uapi/asm/kvm.h
+++ b/tools/arch/x86/include/uapi/asm/kvm.h
@@ -190,7 +190,7 @@ struct kvm_msrs {
 
 /* for KVM_GET_MSR_INDEX_LIST */
 struct kvm_msr_list {
-	__u32 nmsrs; /* number of msrs in entries */
+	__u32 nmsrs; /* number of msrs in indices */
 	__u32 indices[0];
 };
 
diff --git a/tools/testing/selftests/kvm/lib/x86_64/processor.c b/tools/testing/selftests/kvm/lib/x86_64/processor.c
index a8906e60a108..e676fe40bfe6 100644
--- a/tools/testing/selftests/kvm/lib/x86_64/processor.c
+++ b/tools/testing/selftests/kvm/lib/x86_64/processor.c
@@ -714,7 +714,7 @@ uint64_t kvm_get_feature_msr(uint64_t msr_index)
  *
  * Return: KVM CPUID (KVM_GET_CPUID2)
  *
- * Set the VCPU's CPUID.
+ * Get the VCPU's CPUID.
  */
 struct kvm_cpuid2 *vcpu_get_cpuid(struct kvm_vm *vm, uint32_t vcpuid)
 {
